ecaudit/src/main/java/com/ericsson/bss/cassandra/ecaudit/config/AuditYamlConfig.java

```diff
@@ -48,16 +55,21 @@ boolean isFromFile()
      *
      * @return the list of whitelisted users
      */
-    public List<String> getWhitelist()
+    List<String> getWhitelist()
     {
         return whitelist != null ? Collections.unmodifiableList(whitelist) : DEFAULT_WHITELIST;
     }
 
-    /**
-     * @return the SLF4J configuration or {@code null} if not configured.
-     */
-    public AuditYamlSlf4jConfig getSlf4j()
+    ParameterizedClass getLoggerBackendParameters()
     {
-        return slf4j;
+        return logger_backend != null ? deepToStringCopy(logger_backend) : deepToStringCopy(DEFAULT_LOGGER_BACKEND);
+    }
+
+    private ParameterizedClass deepToStringCopy(ParameterizedClass original)
+    {
+        Map<String, String> allStringParameters = original.parameters.entrySet().stream()
+        .collect(Collectors.toMap(Map.Entry::getKey, e -> ((Object)e.getValue()).toString()));
```

Comment:
This is needed because the depending on if user set value with our without hyphen, the parser will make this value a String or an Integer. Futher, the ParameterizedClass instance is anything but immutable. I'll add a comment as to why I'm doing this.

I'm open to suggestions if you know of a more elegant way of doing this.